### DISCUSSION TITLE
🚀 Solo Mining Tutorial for Windows: Start Mining CCX in 3 Easy Steps!

### DISCUSSION THREAD

**Discussion by nathanccxv** [CyberChainXyz/go-cyberchain#3] [Mining]
Hey miners! 👋 

Want to start mining CCX on your Windows PC? Here's a super simple guide to get you up and running with solo mining on CyberChain! 

## 🔧 Step 1: Add CyberChain Network to MetaMask

1. Visit [CyberChain website](https://cyberchain.xyz)
2. Click the "Add to MetaMask" button
3. Switch to CyberChain network in MetaMask
4. Copy **YOUR EVM address** from MetaMask

## 🖥️ Step 2: Run CyberChain Node

1. Download `ccx-Windows-X64-v1.0.0-beta.zip` from [go-cyberchain releases](https://github.com/CyberChainXyz/go-cyberchain/releases)
2. Extract the zip file
3. Edit `run_for_mining.bat`:
   - Replace the placeholder(**0x123...fff**) with **YOUR EVM address**
4. Run `run_for_mining.bat` and wait for sync

## ⛏️ Step 3: Start Mining

1. Download `xMiner-Windows-X64-v0.1.15-beta.zip` from [xMiner releases](https://github.com/CyberChainXyz/xMiner/releases)
2. Extract the zip file
3. Once node is synced, run `solo_mining.bat`

That's it! You're now mining CCX! 🎉

### 💡 Tips
- Make sure your GPU drivers are up to date
- Keep both the node and miner running
- Join our community for support and updates

Happy mining! 🚀

**Comment by 3dpasspool** [comment]
Currently, it seems that Intel graphics cards are not supported. Can you update to support them

**Reply by nathanccxv** [reply]
By default, xMiner will only select AMD and NVIDIA GPUs. However, you can modify the **solo_mining.bat** (or solo_mining.sh on Linux) startup script to allow xMiner to support all OpenCL devices. Simply change the line:

`xMiner.exe -pool=ws://127.0.0.1:8546`

to:

`xMiner.exe -pool=ws://127.0.0.1:8546 -all`

You can give it a try, but I can't guarantee that it will run smoothly on all devices. Looking forward to hearing your results!

**Reply by 3dpasspool** [reply]
thank you

**Reply by 3dpasspool** [reply]
Yes, but the hashrate is very low

**Reply by qusai1991234** [reply]
can we use hiveos ??

**Reply by nathanccxv** [reply]
The HiveOS tutorial has just been released. Your feedback is welcome!
https://github.com/orgs/CyberChainXyz/discussions/8

**Reply by qusai1991234** [reply]
guys any update about the project, they still working on it or just leave it
